### DISCUSSION TITLE
Align button

### DISCUSSION THREAD

**Discussion by Kajmoswega** [mechatroner/vscode_rainbow_csv#166] [Ideas]
Hi, it would be nice to customize the align button. Like setting the size of the spaces which i prefer. Maybe that feature exists, but I can't find it :D

**Comment by mechatroner** [comment]
Hi, sorry for the late reply. Currently, the "Align" function inserts the minimum number of space characters required to align the columns (which is different for each column and depends on the maximum size of entries in that column, and there is also a special rule for integer and float columns). So, decreasing that number would break the alignment. But it is possible to increase the number of space characters above the minimum number, is this what you are proposing?
